packages/reporters/index.js

```diff
@@ -54,8 +54,8 @@ async function run(
 
   // Generate EMP properties to inform monitor modules of important info like token symbols and price identifier.
   const empProps = {
-    collateralCurrencySymbol: await collateralToken.symbol(),
-    syntheticCurrencySymbol: await syntheticToken.symbol(),
+    collateralSymbol: await collateralToken.symbol(),
```

Comment:
these changes were again made for consistency through the repo.

Reply:
Is it time to deprecate this package?

Reply:
ye! separate PR though 🥇